frontend/src/views/admin/assignments/offer-actions.js

```diff
@@ -61,39 +61,39 @@ function ConfirmWithDialogActionButtons(props) {
         setOfferForAssignmentRejected,
     } = props;
 
-    function createOffers() {
+    async function createOffers() {
         for (const assignment of selectedAssignments) {
-            offerForAssignmentCreate(assignment);
+            await offerForAssignmentCreate(assignment);
```

Comment:
No. We do not want to wait one by one. We wan them to all happen in parallel. You can use `Promise.all` for this